protostar/compiler/compiled_contract_reader.py

```diff
@@ -16,13 +16,13 @@ def load_abi_from_contract_path(self, contract_path: Path) -> Optional[AbiType]:
         abi_path = self._get_abi_path_from_contract_path(contract_path)
         if not abi_path.exists():
             return None
-        return self._load_abi(abi_path)
+        return self.load_abi(abi_path)
```

Comment:
Shouldn't we first check for the explicitly specified abi and only if there's none, fall back to the "default" one?

Reply:
We do this in the Call command. This class is unused. I removed it in this PR https://github.com/software-mansion/protostar/pull/1382